.rultor.yml

```diff
@@ -75,3 +75,6 @@ release:
   commanders:
   - original-brownbear
   - yegor256
+docker:
```

Comment:
@dalifreire this has nothing to do with the task, this section of the `.rultor.yml` describes the docker container used to build Rultor itself.
See [here](http://doc.rultor.com/reference.html) for the details on this section and the config file in general.